apps/condo/lang/ru/ru.json

```diff
@@ -70,6 +70,8 @@
   "example.Email": "name@example.org",
   "example.Name": "Иванов Сергей",
   "example.Phone": "+7 (900) 800 22 22",
+  "HotReload.title": "Обновление системы!",
+  "HotReload.message": "Мы обновили систему, поэтому данная страница сейчас перезагрузится",
```

Comment:
```suggestion
  "HotReload.message": "Вышло обновление Doma.ai, страница сейчас будет обновлена",
```

Reply:
Will be removed after conversation with Max